lib/api/cln/cln_client.dart

```diff
@@ -191,4 +193,66 @@ class CLNApi extends AppApi {
             jsonResponse as Map<String, dynamic>,
             snackCase: !mode.withCamelCase()));
   }
+
+  @override
+  Future<AppGenerateInvoice> generateInvoice(String label, String description,
+      {int? amount}) {
+    dynamic params;
+    switch (mode) {
+      case ClientMode.grpc:
+        if (amount == null) {
+          params = CLNGenerateInvoiceRequest(
+              grpcRequest: InvoiceRequest(
+                  label: label,
+                  description: description,
+                  amountMsat: AmountOrAny(any: true)));
+        } else {
+          Amount amount2 = Amount();
+          amount2.msat = Int64(amount);
+          params = CLNGenerateInvoiceRequest(
+              grpcRequest: InvoiceRequest(
+                  label: label,
+                  description: description,
+                  amountMsat: AmountOrAny(
+                      amount: Amount(msat: amount2.msat), any: false)));
+        }
+        break;
+      case ClientMode.unixSocket:
+        if (amount == null) {
+          params = CLNGenerateInvoiceRequest(unixRequest: <String, dynamic>{
+            'amount_msat': 'any',
+            'label': label,
+            'description': description
+          });
+        } else {
+          params = CLNGenerateInvoiceRequest(unixRequest: <String, dynamic>{
+            'amount_msat': amount,
+            'label': label,
+            'description': description
+          });
+        }
+        break;
+      case ClientMode.lnlambda:
+        if (amount == null) {
+          params = CLNGenerateInvoiceRequest(unixRequest: <String, dynamic>{
+            'amount_msat': 'any',
+            'label': label,
+            'description': description
+          });
+        } else {
+          params = CLNGenerateInvoiceRequest(unixRequest: <String, dynamic>{
+            'amount_msat': amount,
+            'label': label,
+            'description': description
+          });
+        }
```

Comment:
```suggestion
          params = CLNGenerateInvoiceRequest(unixRequest: <String, dynamic>{
            'amount_msat': amount == null ? 'any' : amount,
            'label': label,
            'description': description
          });
```

I think this form save some code